src/Avalonia.FuncUI.DSL/Base/Control.fs

```diff
@@ -31,4 +33,5 @@ module Control =
         static member contextMenu<'t when 't :> Control>(menu: ContextMenu) : IAttr<'t> =
             AttrBuilder<'t>.CreateProperty<ContextMenu>(Control.ContextMenuProperty, menu, ValueNone)
            
-           
\ No newline at end of file
+        static member key<'t when 't :> Control>(key: string) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<string>(ViewMetaData.KeyProperty, key, ValueNone)
```

Comment:
Having UUIDs is common for items in a list - providing an overload would be handy.
```suggestion
        static member key<'t when 't :> Control>(key: string) : IAttr<'t> =
            AttrBuilder<'t>.CreateProperty<string>(ViewMetaData.KeyProperty, key, ValueNone)
            
        static member key<'t when 't :> Control>(key: System.Guid) : IAttr<'t> =
            AttrBuilder<'t>.CreateProperty<string>(ViewMetaData.KeyProperty, string key, ValueNone)
```